actors/builtin/miner/policy.go

```diff
@@ -196,6 +196,11 @@ const DealLimitDenominator = 134217728 // PARAM_SPEC
 // for permissioned actor methods and winning block elections.
 const ConsensusFaultIneligibilityDuration = ChainFinality
 
+// PoStChallengePeriod is the period after a challenge window ends during which
+// PoSts submitted during that period may be challenged.
+// TODO: this name can easily be confused with WPoStChallengeWindow.
+const WPoStProofChallengePeriod = 2 * ChainFinality // PARAM_TODO
```

Comment:
Please add an `init` assertion that this is less than the deadline window.

Edit: I see you added a test instead. We already have some assertions above. I think it makes sense to keep them together, and in this file. But centralising them in either place is ok.

Reply:
I've moved these constraints into the init function.